stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/StreamAttachmentFactories.kt

```diff
@@ -0,0 +1,32 @@
+package io.getstream.chat.android.compose.ui.attachments
+
+import androidx.compose.runtime.Composable
+import io.getstream.chat.android.client.models.Attachment
+import io.getstream.chat.android.compose.state.messages.attachments.AttachmentState
+
+public object StreamAttachmentFactories {
+
+    /**
+     * Default attachment factories we provide, which can transform image, file and link attachments.
+     *
+     * Uses the functions below to display the UI.
+     * */
+    public val defaultFactories: List<AttachmentFactory> = listOf(
+        LinkAttachmentFactory(),
+        GiphyAttachmentFactory(),
```

Comment:
Perhaps `Giphy` should go first here, to catch things before `LinkAttachmentFactory` would get involved. Then we don't need the `Giphy` check in that factory (though we might still want to keep it, just in case)

Reply:
I think relying on the ordering is not the best approach, because we never know what the BE team does to the data - e.g. this whole change with the `titleLink`.

This way we can check for links, but unfortunately giphy also has links (even though it could use other properties), so we have to add this check.